lib/error/include/hse/error/merr.h

```diff
@@ -114,10 +109,15 @@ size_t
 merr_strerror(merr_t err, char *buf, size_t buf_sz);
 
 /**
- * merr_strinfo() - Format file, line, and errno from merr_t
+ * merr_strinfo() - Format file, line, ctx, and errno from merr_t
  */
 char *
-merr_strinfo(merr_t err, char *buf, size_t buf_sz, size_t *need_sz);
+merr_strinfo(
+    merr_t err,
+    char *buf,
+    size_t buf_sz,
+    merr_stringify stringify_ctx,
+    size_t *need_sz);
```

Comment:
Isn't "stringify_ctx"  just a function pointer?  Why call it a context?

Reply:
"stringify the context from merr_ctx()" is conceptually what it means

Reply:
Oh, I see, it's a sentence, as in "please stringify this context", but reads as a "strignify" context.  ctx_stringify() would make more sense to me, especially if we grow more actions (e.g., ctx_format(), ctx_print(), whatever...)